pkg/utils/tags.go

```diff
@@ -82,3 +82,33 @@ func RemoveTags(obj interface{}, tags []string) error {
 	structTags.Set(res)
 	return nil
 }
+
+func HasTags(obj interface{}, tags []string) bool {
```

Comment:
The use of `reflect` here and throughout these utilities is rather hideous. We control these object definitions and really should have some proper Tag-related methods and interfaces defined in https://github.com/Kong/go-kong/blob/main/kong/consumer.go and such.

However, it probably gets a pass here due to the existing pattern :frowning_face:

Reply:
+1 to what @rainest said.

Also I'd add a godoc here so that we're clear if we're checking all tags or at least one of the provided tags.